cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -150,7 +151,15 @@ func (s *storageExt) Start(_ context.Context, host component.Host) error {
 		if err != nil {
 			return fmt.Errorf("failed to initialize storage '%s': %w", storageName, err)
 		}
-		s.factories[storageName] = factory
+		storageMetricsFactory := mf.Namespace(
+			metrics.NSOptions{
+				Name: "storage",
+				Tags: map[string]string{
+					"kind": storageName,
```

Comment:
"kind" should be something like "cassandra" (which you can save from the switch statement above.
```suggestion
					"name": storageName,
					"kind": storageKind,
```